src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/SentenceCalculation.java

```diff
@@ -52,8 +63,13 @@
     @JoinColumn(name = "STAFF_ID")
     private Staff staff;
 
-    @Column(name = "CALC_REASON_CODE")
-    private String reasonCode;
+    @ManyToOne
+    @NotFound(action = IGNORE)
```

Comment:
Are there missing entries?

Reply:
Good point! There shouldn't be so removed that annotation